.buildkite/pipeline_perf.py

```diff
@@ -11,31 +11,31 @@
         "label": "🖴 Block Performance",
         "test_path": "integration_tests/performance/test_block_performance.py",
         "devtool_opts": "-r 16834m -c 1-10 -m 0",
-        "timeout_in_minutes": 240,
+        "timeout_in_minutes": 960,
```

Comment:
I think this could be done below, to make it depend on the number of retries:

around line 76

```python
test["timeout_in_minutes"] *= (args.retries + 1)
```

Reply:
oh, yeah, that's a better way. Forgot retries were adjustable

Reply:
Done :)